src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigPathsControl.xaml.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Threading;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+using Microsoft.VisualStudio.PlatformUI;
+using NuGet.VisualStudio;
+using NuGet.PackageManagement.Telemetry;
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigPathsControl.xaml
+    /// </summary>
+    public partial class ConfigPathsControl : UserControl
+    {
+        public ConfigPathsWindowViewModel ConfigPathsWindow { get; set; }
+        public ConfigPathsViewModel SelectedPath { get; set; }
+        public ICommand OpenConfigurationFile { get; set; }
+
+        public ConfigPathsControl()
+        {
+            ConfigPathsWindow = new ConfigPathsWindowViewModel();
+            OpenConfigurationFile = new DelegateCommand(ExecuteOpenConfigurationFile, IsSelectedPath, NuGetUIThreadHelper.JoinableTaskFactory);
+            DataContext = this;
+            InitializeComponent();
+        }
+
+        private bool IsSelectedPath(object obj)
+        {
+            return _configurationPaths.SelectedItem != null;
+        }
```

Comment:
The parameter `obj` is unused. Is that expected? If yes, do we need that parameter? If not, can we just remove it.